cmake/oidn_bnns.cmake

```diff
@@ -3,7 +3,7 @@
 
 if(NOT IOS)
   set(SDK_VERSION_COMMAND  xcrun -sdk  macosx --show-sdk-version)
-  set(SDK_TARGET 11.0)
+  set(SDK_TARGET 14.0)
```

Comment:
We cannot raise the SDK target on macOS. This doesn't seem to be relevant for iOS support.

Reply:
removing this, uncaught remnant from the debug process.